pyEpiabm/pyEpiabm/tests/test_utility/test_state_transition_matrix.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from enum import Enum
+import pandas as pd
+import numpy as np
+from pandas.testing import assert_frame_equal
+
+import pyEpiabm as pe
+from pyEpiabm.property.infection_status import InfectionStatus
+from pyEpiabm.utility import StateTransitionMatrix
+
+
+class TestStateTransitionMatrix(unittest.TestCase):
+    """Test the 'StateTransitionMatrix' class.
+    """
+    def test_build_initial_matrix(self):
+        """Tests the build_state_transition_matrix method by asserting if it is
+        equal to the initial matrix expected.
+        """
+        matrix_object = StateTransitionMatrix()
+        matrix = matrix_object.build_state_transition_matrix()
+        labels = [status.name for status in InfectionStatus]
+        zero_filled_dataframe = pd.DataFrame(np.zeros((10, 10)),
```

Comment:
maybe replace 10 with len(InfectionStatus) so this still passes even if a new InfectionStatus is added